akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -1258,11 +1258,11 @@ private[akka] class DDataShardCoordinator(
 
   private val stateReadConsistency = settings.tuningParameters.coordinatorStateReadMajorityPlus match {
     case Int.MaxValue => ReadAll(settings.tuningParameters.waitingForStateTimeout)
-    case additional   => ReadMajorityPlus(settings.tuningParameters.waitingForStateTimeout, majorityMinCap, additional)
+    case additional   => ReadMajorityPlus(settings.tuningParameters.waitingForStateTimeout, additional, majorityMinCap)
   }
   private val stateWriteConsistency = settings.tuningParameters.coordinatorStateWriteMajorityPlus match {
     case Int.MaxValue => WriteAll(settings.tuningParameters.updatingStateTimeout)
-    case additional   => WriteMajorityPlus(settings.tuningParameters.updatingStateTimeout, majorityMinCap, additional)
+    case additional   => WriteMajorityPlus(settings.tuningParameters.updatingStateTimeout, additional, majorityMinCap)
```

Comment:
For context, here is where the case class is defined:

https://github.com/akka/akka/blob/de7d18667f014f707cebafcb7acac8062fb9e255/akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala#L487-L488

Reply:
Btw, when the values are used inside `Replicator` we do respect the correct order. So, the impl is correct. Only the initialization is swapped.